chipsec/utilcmd/txt_cmd.py

```diff
@@ -0,0 +1,171 @@
+# CHIPSEC: Platform Security Assessment Framework
+# Copyright (c) 2010-2021, Intel Corporation
```

Comment:
Please remove Intel's copy right.

Reply:
Done. I am not familiar with the rules regarding the copyright comments in the files, and copied the header from other files in `chipsec/utilcmd/` to be consistent.
I removed `# Copyright (c) 2010-2021, Intel Corporation` while keeping the GPL2 license information.

Reply:
That's perfect! License should be consistent for the project. You're welcome to include your own copyright, but not needed.